templates/ajaxform.html5

```diff
@@ -22,82 +22,78 @@
 </div>
 </form>
 
-<?php if ($this->jquery): ?>
 <script>
-(function($) {
-"use strict";
-    $(document).ready(function() {
-        var form = $(document.querySelector('input[name="FORM_SUBMIT"][value="<?php echo $this->formSubmit; ?>"]').form);
-        form.submit(function() {
-            $.ajax({
-                type: '<?php echo strtoupper($this->method); ?>',
-                data: window.FormData ? new FormData(this) : form.serialize(),
-                cache: false,
-                contentType: false,
-                processData: false,
-                success: function(data) {
-                    if (data == 'true') {
-                        form.off('submit').submit();
-                        return;
-                    }
-                    form.animate({'opacity':0}, 200, 'swing', function() {
-                        form.html(data);
-                        form.animate({'opacity':1}, 200);
-
-                        try {
-                            form.find('input,textarea').each(function (el) {
-                                el.cleardefault();
-                            });
-                        } catch (err) {}
-
-                        form.trigger('ajax_change');
-                        $(window).trigger('ajax_change');
-                    });
-                }
+(function() {
+    var triggeredButton = null;
+    var form = document.querySelector('input[name="FORM_SUBMIT"][value="<?= $this->formSubmit; ?>"]').form;
+    if (!form) {
+        return;
+    }
+
+    function request(method, uri, body, callback) {
+        var body = body || null;
+        var request = new XMLHttpRequest();
+        request.open(method, uri, true);
+        request.setRequestHeader('X-Requested-With', 'XMLHttpRequest');
+        request.setRequestHeader('Contao-Ajax-Form', '<?= $this->formSubmit; ?>');
+
+        request.onload = function() {
+            callback.apply(this);
+
+            var event = new Event('ajax_change');
+            form.dispatchEvent(event);
+            window.dispatchEvent(event);
+        };
+        request.send(body)
+    }
+
+    function addButtonEvents(form) {
+        var buttons = form.querySelectorAll('input[type="submit"],button[type="submit"]');
+        for (var i=0;i<buttons.length;i++) {
+
+            buttons[i].addEventListener('click', function (e) {
+                triggeredButton = e.target;
```

Comment:
I think `e.currentTarget` would be more appropriate here also because of the fact that a submit `<button>` can have some HTML inside.

> The currentTarget refers to the element that the event listener directly attached to while the target still refers to the specific element we clicked.